pkgs/applications/misc/waybar/sysconfdir.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/meson.build b/meson.build
+index 9bf5c43..fa8bdbd 100644
+--- a/meson.build
++++ b/meson.build
+@@ -128,7 +128,7 @@ executable(
+ install_data(
+     './resources/config',
+     './resources/style.css',
+-    install_dir: join_paths(get_option('out'), 'etc/xdg/waybar')
```

Comment:
This is actually correct and we would want [more applications](https://github.com/hughsie/fwupd/pull/824) to do this.

You should add `"-Dout=${placeholder "out"}"` and `"--sysconfdir=/etc"` to `mesonFlags`.

Reply:
Thanks for the hint. Why do I need ```"--sysconfdir=/etc"```? Unless the ```sysconfdir``` is passed by meson to the applications, settings this should not make a difference.

Reply:
Yup, in this case sysconfdir will probably have no effect, as they use `g_get_system_data_dirs` to find the configuration.